.github/workflows/wheels.yml

```diff
@@ -52,24 +52,26 @@ jobs:
           CIBW_TEST_REQUIRES: absl-py pytest pytest-xdist
           CIBW_TEST_COMMAND: pytest -n auto {project}
 
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@834a144ee995460fba8ed112a2fc961b36a5ec5a # ratchet: actions/upload-artifact@v4
         with:
           path: ./wheelhouse/*.whl
+          overwrite: true
```

Comment:
Note: with v3 of the upload-artifact action, it effectively set `overwrite: true` by default. I found that after upgrading to v4, we needed to explicitly set this or the job would fail, because each run creates identically-named artifacts.